src/vm/object.h

```diff
@@ -2678,7 +2678,6 @@ class StringBufferObject : public Object
 
     static void ReplaceBuffer(STRINGBUFFERREF *thisRef, __in_ecount(newLength) WCHAR *newBuffer, INT32 newLength);
     static void ReplaceBufferAnsi(STRINGBUFFERREF *thisRef, __in_ecount(newCapacity) CHAR *newBuffer, INT32 newCapacity);    
```

Comment:
Are these guys dead too? I can't see a caller to ReplaceBufferAnsi only ReplaceBufferAnsiInternal. Unless some macro magic measn they're the same.

Reply:
They are called from here: https://github.com/dotnet/coreclr/blob/master/src/classlibnative/bcltype/stringbuffer.cpp#L29

Reply:
Uh, I forgot that there is a world outside of /vm/ that uses this header.